x-pack/solutions/security/plugins/security_solution/server/lib/detection_engine/rule_types/esql/wrap_esql_alerts.test.ts

```diff
@@ -53,26 +34,18 @@ describe('wrapSuppressedEsqlAlerts', () => {
 
   test('should call generateAlertId for alert id', () => {
     jest.spyOn(esqlUtils, 'generateAlertId').mockReturnValueOnce('mocked-alert-id');
-    const completeRuleCloned = cloneDeep(completeRule);
-    completeRuleCloned.ruleParams.alertSuppression = {
-      groupBy: ['someKey'],
-    };
+    const newSharedParams = getSharedParamsMock({
```

Comment:
Why not just 
```suggestion
    const sharedParams = getSharedParamsMock({
```

?

Reply:
There's also a `sharedParam` at the outer scope that I wanted to avoid shadowing